utils.py

```diff
@@ -104,6 +104,13 @@ def publish(temperatures, battery, client, base_topic, device_name):
 
     client.publish("{0}/{1}/battery".format(base_topic, device_name), battery)
 
+def publishPulse2000(temperatures, battery, main_element, client, base_topic, device_name):
```

Comment:
Instead of creating a separate method for the Pulse2000 I would add the main_element as an optional parameter to the publish method, and do a quick if check to see if it should publish the value. This way we can also refactor to make battery optional if we want to.
https://www.geeksforgeeks.org/default-arguments-in-python/